src/replaykit.cs

```diff
@@ -136,6 +136,21 @@ interface RPScreenRecorder {
 		[Async]
 		[Export ("stopRecordingWithOutputURL:completionHandler:")]
 		void StopRecording (NSUrl url, [NullAllowed] Action<NSError> completionHandler);
+
+		[Async]
+		[NoTV, NoWatch, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("exportClipToURL:duration:completionHandler:")]
+		void ExportClip (NSUrl url, double duration, [NullAllowed] Action<NSError> completionHandler);
```

Comment:
I'm hesitant about changing this ones name from ExportClipToUrl to ExportClip. 
NSUrl is the first argument, but does this name change make it more confusing to use?

Reply:
When in doubt, you can check the Swift naming, in this case it's `exportClip` (https://developer.apple.com/documentation/replaykit/rpscreenrecorder/3752130-exportclip), so I think this is totally fine.

Reply:
Makes sense! Thanks!